api/comments/views.py

```diff
@@ -0,0 +1,140 @@
+from modularodm import Q
+from modularodm.exceptions import NoResultsFound
+from rest_framework import generics, permissions as drf_permissions
+from rest_framework.exceptions import NotFound, ValidationError, PermissionDenied
+from api.comments.serializers import CommentSerializer, CommentDetailSerializer, CommentReportsSerializer, CommentReportDetailSerializer, CommentReport
```

Comment:
Small request, but please change the order of these imports to be compliant with PEP8. It should always be obvious where to look to see if something has already been imported.

https://www.python.org/dev/peps/pep-0008/#imports
>  Imports should be grouped in the following order:
standard library imports
related third party imports
local application/library specific imports

> You should put a blank line between each group of imports. 

Within those subgrouping, keeping things alphabetical is handy also.